conf/application.conf

```diff
@@ -24,6 +24,7 @@ mongodb {
 vaults {
   settings-path="/Users/localhome/Desktop/vaults"
   settings-path=${?VAULT_SETTINGS_PATH}
+  vaultDataPath="/opt/vault_data"
 }
```

Comment:
please put in `vaultDataPath=${VAULT_DATA_PATH}` under this - it allows us to override it with an environment variable if we want to

Reply:
I have added this as 'vaultDataPath=${?VAULT_DATA_PATH}' as I believe this will not cause an error it VAULT_DATA_PATH does not exist.